WalletWasabi/Models/EffectiveTransactionStatus.cs

```diff
@@ -0,0 +1,19 @@
+using System.Collections.Generic;
+using NBitcoin;
+
+namespace WalletWasabi.Models;
+
+public record Ancestor
+{
+	public uint256 txid { get; set; }
+	public long fee { get; set; }
+	public long weight { get; set; }
+}
+
+public record EffectiveTransactionStatus
```

Comment:
It is hard for me to come with a good name for this but the it is clear that this is not an status and that _effective transaction_ is not a concept. If we cannot define it then I would go with the semantic given by the client and use something like `CpfpInfo`